src/components/FileManager/UploadFileArea.tsx

```diff
@@ -19,7 +19,7 @@ import { NativeTypes } from 'react-dnd-html5-backend'
 import styled, { css } from 'styled-components'
 
 import { Upload } from './FileManager'
-import { Action, actions } from './FileSectionState'
+import { Action, actions, State } from './FileSectionState'
```

Comment:
Looks like State is imported, but it's never used.

Reply:
State removed from import....